config/logging_level.go

```diff
@@ -26,13 +27,18 @@ var globalLogger struct {
 func InitLoggingSettings(logger logging.Logger, cmdLineDebugFlag bool) {
 	globalLogger.logger = logger
 	globalLogger.cmdLineDebugFlag = cmdLineDebugFlag
+	gologLogger := golog.NewDebugLogger("init")
```

Comment:
Q: does `"init"` have any special meaning or function?

Reply:
Under the hood, this name is used by Zap Named ([code](https://github.com/edaniels/golog/blob/28954395e8d0e7ae7f0046e4c845d553b94e4805/logger.go#L163C2-L163C8)).

I updated the name and it will show like this 
```
2024-05-14T18:29:43.531-0400	DEBUG	robot_server.golog   operation/web.go:59	unchecked error	{"error": "rpc error: code = Internal desc = transport: SendHeader called multiple times"}
```